src/selectors/getTaskListToShowSelector.ts

```diff
@@ -8,6 +8,7 @@ export const getTaskListToShowSelector: Selector<ToDoState, TodoToShow[]> = (
     return {
       id: task.id,
       task: task.task,
+      createdAt: format(task.createdAt, 'd.L.yyyy H:mm'),
```

Comment:
for now, that is the only file where I use this pattern for format() 
but also I have another pattern for datePicker input 
should I create separate file for those or it will be enough just to make constant variables on each

Reply:
I think it's better to create